lib/kuzzle/kuzzle.ts

```diff
@@ -252,7 +292,7 @@ class Kuzzle extends KuzzleEventEmitter {
    *
    * @returns {Promise<void>}
    */
-  async install (installations) {
+  async install (installations: Array<InstallationConfig>): Promise<void> {
```

Comment:
Prefer this notation (more simple)
```suggestion
  async install (installations: InstallationConfig[]): Promise<void> {
```